client/src/planwise/client/scenarios/edit.cljs

```diff
@@ -163,7 +163,7 @@
    [m/Fab {:class-name "btn-floating"
            :on-click #(dispatch [:scenarios.new-action/toggle-options])}
     (cond computing? "stop"
-          (or (= state :new-provider) (= state :new-intervention)) "cancel"
+          (or (= state :new-provider) (= state :new-intervention) (= state :new-suggestion)) "cancel"
```

Comment:
There are a couple of places where this or a similar pattern is repeated. I suggest extracting the predicate to a function (eg. `state-cancellable?` maybe).